### PR TITLE
Mark 777 use single generator for marketplace and cortex

### PR DESCRIPTION
**Pull request by kpradhan-cs**
This modifies the existing codes to add the files required by the marketplace.
There is another pull request which uses separate codes to add those files. We need to merge only one.

The other pull request is https://github.com/CognitiveScale/generator-cortex/pull/47.

### CODE REVIEW

File: generators/datasets/templates/scripts/win32/build-dataset.bat [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+SET SCRIPT_DIR=%SCRIPT_DIR:~0,-1%
+SET SOURCE_DIR=%SCRIPT_DIR%
+SET BUILD_DIR=%SCRIPT_DIR%\build
+SET BUILD_FILE=%BUILD_DIR%\dataset.zip
+
+if not exist %BUILD_DIR%/nul mkdir "%BUILD_DIR%"
+
+echo Creating build in %BUILD_DIR%...
+powershell %SCRIPT_DIR%/buildzip.ps1 %SOURCE_DIR% %BUILD_FILE%
+echo Created function.zip in %BUILD_DIR%
```
Comment (daldridge-cs) [inline generators/datasets/templates/scripts/win32/build-dataset.bat]: Copy/paste error.  Should say `dataset.zip`

File: generators/skill/templates/daemon/scripts/common/win32/build-skill.bat [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+SET SCRIPT_DIR=%SCRIPT_DIR:~0,-1%
+SET SOURCE_DIR=%SCRIPT_DIR%
+SET BUILD_DIR=%SCRIPT_DIR%\build
+SET BUILD_FILE=%BUILD_DIR%\skill.zip
+
+if not exist %BUILD_DIR%/nul mkdir "%BUILD_DIR%"
+
+echo Creating build in %BUILD_DIR%...
+powershell %SCRIPT_DIR%/buildzip.ps1 %SOURCE_DIR% %BUILD_FILE%
+echo Created function.zip in %BUILD_DIR%
```
Comment (daldridge-cs) [inline generators/skill/templates/daemon/scripts/common/win32/build-skill.bat]: Copy/paste error.  Should be `skill.zip`

File: generators/skill/templates/function/scripts/common/win32/build-skill.bat [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+SET SCRIPT_DIR=%SCRIPT_DIR:~0,-1%
+SET SOURCE_DIR=%SCRIPT_DIR%
+SET BUILD_DIR=%SCRIPT_DIR%\build
+SET BUILD_FILE=%BUILD_DIR%\skill.zip
+
+if not exist %BUILD_DIR%/nul mkdir "%BUILD_DIR%"
+
+echo Creating build in %BUILD_DIR%...
+powershell %SCRIPT_DIR%/buildzip.ps1 %SOURCE_DIR% %BUILD_FILE%
+echo Created function.zip in %BUILD_DIR%
```
Comment (daldridge-cs) [inline generators/skill/templates/function/scripts/common/win32/build-skill.bat]: Copy/paste error.  This should be `skill.zip`.

File: generators/skill/templates/daemon/scripts/common/win32/build-skill.bat (hunk 2 above)
Comment (mchutani-CS) [reply generators/skill/templates/daemon/scripts/common/win32/build-skill.bat]: Fixed.

File: generators/datasets/templates/scripts/win32/build-dataset.bat (hunk 1 above)
Comment (mchutani-CS) [reply generators/datasets/templates/scripts/win32/build-dataset.bat]: Fixed.

File: generators/skill/templates/function/scripts/common/win32/build-skill.bat (hunk 3 above)
Comment (mchutani-CS) [reply generators/skill/templates/function/scripts/common/win32/build-skill.bat]: Fixed.

**Comment by mchutani-CS** [comment]
> I tried this out and couldnt get the marketplace generator to work:
> 
> ```
> cortex marketplace connections generate connections/mongo/connection.yaml  --yaml
> You don't seem to have a generator with the name “@c12e/cortex:marketplace_connection” installed.
> ```
> 
> Also, daemon Dockerfile is using c12e/cortex-python-lib:latest-dev which doesnt seem like a great idea for customers

@cdraper-CS The MP generators are merged with the base cortex generators. 
So the following command should generate marketplace related files as well:
`cortex connections generate`

**Comment by mchutani-CS** [comment]
> > I tried this out and couldnt get the marketplace generator to work:
> > ```
> > cortex marketplace connections generate connections/mongo/connection.yaml  --yaml
> > You don't seem to have a generator with the name “@c12e/cortex:marketplace_connection” installed.
> > ```
> > 
> > 
> > Also, daemon Dockerfile is using c12e/cortex-python-lib:latest-dev which doesnt seem like a great idea for customers
> 
> @cdraper-CS The MP generators are merged with the base cortex generators.
> So the following command should generate marketplace related files as well:
> `cortex connections generate`

Also, have made some changes. We've decided to merge generators with `cortex <type> generate`. Kishan missed deleting the marketplace specific generators. 
Now the commands `cortex marketplace <type> generate` shouldn't exist. The MP specific files/scripts should be generated along with the original generators `cortex <type> generate`.

**Comment by mchutani-CS** [comment]
> The daemon generator with python2 and nodejs generates functions not daemons

With python2, I can see it's generating a daemon (not a function).
With nodejs, I agree it's generating a function, but I believe that was the case even before this PR. So should we fix that in a separate branch with having a bug ticket associated with it?

**Comment by cdraper-CS** [comment]
@mchutani-CS  Can you fix it so that the 'cortex marketplace xxx generate' options don't appear as valid commands for the user? (i.e. if you do `cortex marketplace connections`, `generate` should not be in the list.

**Pull request merged**